templates/case-studies/templates/_macros-text.jinja

```diff
@@ -0,0 +1,15 @@
+# Params
+# title: Title of text section (required)
+# description: Description of text section (required)
+
+{%- macro text(title, description) -%}
+  <section class="p-strip is-deep">
+    <div class="row">
+      <div class="col-start-large-4 col-9">
+        <hr class="p-rule"/>
+        <h2 class="u-sv3">{{ title }}</h2>
```

Comment:
If the spacing on the figma is correct, looks like this should be wrapped in a shallow section div rather than using the u-sv utility class on the heading
![image](https://github.com/user-attachments/assets/82362ba6-fc09-4224-ae10-f9e7e388834a)

Reply:
Thanks for the catch! I've applied the fixes as suggested